runtime/hydradx/src/weights/pallet_scheduler.rs

```diff
@@ -98,12 +98,12 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for HydraWeight<T> {
 	/// The range of component `s` is `[128, 4194304]`.
 	fn service_task_fetched(s: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `246 + s * (1 ±0)`
+		//  Measured:  `279 + s * (1 ±0)`
 		//  Estimated: `4197809`
-		// Minimum execution time: 26_513_000 picoseconds.
-		Weight::from_parts(26_659_000, 3711)
-			// Standard Error: 6
-			.saturating_add(Weight::from_parts(1_318, 0).saturating_mul(s.into()))
+		// Minimum execution time: 25_030_000 picoseconds.
+		Weight::from_parts(25_294_000, 4197809)
```

Comment:
wow this would brick scheduler again, i thought the benchmark is fixed already @Roznovjak